assets/scss/_variables_project.scss

```diff
@@ -81,14 +81,19 @@ h2, h3 {
       font-weight: $font-weight-bold;
     }
     &__section.active {
-      color: $primary;    
+      color: $primary;
     }
     &__section.selected {
       color: $secondary;
     }
   }
 }
 
+.td-main .td-content {
+  .schedule-table + table td  {
```

Comment:
Is the `td-main` and `td-content` required? If we think the specificity is valuable enough to include, should we include `workflow` from below in there as well?

In terms of naming, given that we have "workflow" which is also a table with a shortcode, should we also align the naming (eg just `schedule` to match the `workflow` example)? Seems worthwhile to consider now before we start using it.

Reply:
cool cool! I've decreased specificity and renamed it, both to match workflow. We can always increase specificity later.